### PR TITLE
Blog Post to Describe Defectdojo Dev Setup on Apple Silicon

### PR DESCRIPTION
**Pull request by Weltraumschaf**

### CODE REVIEW

File: documentation/blog/2024-10-25-run-x86-images-with-kubernetes-on-apple-silicon.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  ./helm/defectdojo \
+  --set django.ingress.enabled=true \
+  --set django.ingress.activateTLS=false \
+  --set createSecret=true \
+  --set createRabbitMqSecret=true \
+  --set createRedisSecret=true \
+  --set createMysqlSecret=true \
+  --set createPostgresqlSecret=true \
+  --set host="defectdojo.default.colima.local" \
+  --set "alternativeHosts={localhost}"
```
Comment (J12934) [inline documentation/blog/2024-10-25-run-x86-images-with-kubernetes-on-apple-silicon.md:158]: the rabbitmq and mysql options aren't avaialble anymore so they don't have any effect anymore

```suggestion
helm upgrade --install \
  defectdojo \
  ./helm/defectdojo \
  --set django.ingress.enabled=true \
  --set django.ingress.activateTLS=false \
  --set createSecret=true \
  --set createRedisSecret=true \
  --set createPostgresqlSecret=true \
  --set host="defectdojo.default.colima.local" \
  --set "alternativeHosts={localhost}"
```

**Review by J12934 — COMMENTED** [review]
looks good, one small note

**Review by J12934 — APPROVED** [review]

**Comment by Weltraumschaf** [comment]
> looks good, one small note

I copy&pasted this. I'll push it onto this PR.

**Comment by J12934** [comment]
website builds fails:

5:15:43 PM:   - Broken link on source page path = /docs/how-tos/persistence-storage:
5:15:43 PM:      -> linking to /blog/2024/10/25/run-x86-images-with-kubernetes-on-apple-silicon

File: documentation/blog/2024-10-25-run-x86-images-with-kubernetes-on-apple-silicon.md (hunk 1 above)
Comment (Weltraumschaf) [reply documentation/blog/2024-10-25-run-x86-images-with-kubernetes-on-apple-silicon.md:158]: Did you already removed this from documentation/docs/how-tos/persistence-storage.md? I think I've copied it from there.

**Comment by Weltraumschaf** [comment]
> website builds fails:
> 
> 5:15:43 PM: - Broken link on source page path = /docs/how-tos/persistence-storage: 5:15:43 PM: -> linking to /blog/2024/10/25/run-x86-images-with-kubernetes-on-apple-silicon

But why? It works on my machine 🤔

**Pull request merged**